qdata/parser.py

```diff
@@ -101,7 +110,13 @@ def statement(self, *args):
         elif args[0] == "include":
             filename = args[1][1:-1]
 
-            with open(filename, "r") as f:
+            p = pathlib.Path(filename)
+
+            if not p.is_file():
+                # use the standard include directory
+                p = pathlib.Path(__file__).parent / "include" / filename
+
+            with open(p, "r") as f:
```

Comment:
Attempts to use the standard `include` directory, _if_ the file is not on the existing path. This will be required for writing tests.